### ISSUE TITLE
Please add a @NgModule annotation.

### ISSUE THREAD

**Issue by alexandruantonica** [henrychavez/nativescript-bottom-navigation#32]
I've tried the last version `1.4.0`. Everything is fine during the build but I had a problem with the app started. I got this error : 
`JS ERROR Error: Unexpected value 'NativescriptBottomNavigationModule' imported by the module 'AppModule'. Please add a @NgModule annotation.` 

I spend some time to understand what is wrong with my app because the `demo` works perfectly without that error.

Do you have any ideas? 

Thanks.

**Comment by henrychavez** [comment]
uhmm can you put your app.module.ts

**Comment by alexandruantonica** [comment]
app.module.ts
```
import { ErrorHandler, NgModule, NO_ERRORS_SCHEMA } from "@angular/core";
import { NativeScriptModule } from "nativescript-angular/nativescript.module";
import { AppRoutingModule } from "./app.routing";
import { AppComponent } from "./app.component";
import { HomeComponent } from "./home/home.component";
import { MyProfilesComponent } from "./my-profiles/my-profiles.component";
import { EditProfileComponent } from "./edit-profile/edit-profile.component";
import { FieldsComponent } from "./common/fields/field.component";
import { ProfileCardComponent } from "./common/profilecard/profilecard.component";
import { AddNewProfileComponent } from "./add-new-profile/add-new-profile.component";
import { ViewProfileComponent } from "./view-profile/view-profile.component";
import { RadListViewComponent } from "nativescript-ui-listview/angular";
import { RavenErrorHandler, ErrorsTracker } from "./services/errors-tracker.service";
import { ErrorTrackerInit } from "./services/errors-tracker.service";
import { EventsTracker, EventsTrackerInit } from "./utils/event-tracker/events-tracker.service"
import { AddNewContactComponent } from "./contacts/add-new-contact/add-new-contact.component";
import { ViewCardComponent } from "./view-profile/view-card/view-card.component";
import { ManualExchangeValidatorService } from "~/core/exchange/manual-exchange-validator.service";
import { CoreModule } from "~/core/core.module";
import { ApiModule } from "~/api-client/api.module";
import { apiConfig } from "~/app.config";
import { CommonService } from "~/services/common.service";
import { ExchangeTapService } from "~/services/exchange-tap.service";
import { ConnectivityService } from "~/services/connectivity.service";
import { UserSettings } from "~/services";
import { LogoutService } from "~/services/logout.service";
import { ImageProcessorService } from "~/services/image-processor.service";
import { MyProfilesDaoService } from "~/my-profiles/my-profiles-dao.service";
import { TabSelectionObserverService } from "~/home/tab-selection-observer.service";
import { UUIDService } from "~/services/uuid.service";
import { TimerModalService } from "~/services/timer-modal.service";
import { WebViewService } from "~/utils/webview/webview.service";
import { BeaconExchangeService } from "~/services/beacon-exchange.service";
import { BeaconScanningService } from "~/services/beacon-scanning.service";
import { BeaconExchangeLogService } from "~/services/beacon-exchange-log.service";
import { FormFields } from "~/services/form-fields.service";
import { PhoneService } from "~/utils/phone/phone.service";
import { DateTimeService } from "~/services/date-time.servce";
import { ProfileDefaultService } from "~/common/profilecard/profile.default.service";
import { registerElement } from "nativescript-angular";
import { NativescriptBottomNavigationModule } from "nativescript-bottom-navigation/angular";

registerElement("CardView", () => require("nativescript-cardview").CardView);
registerElement("PreviousNextView", () => require("nativescript-iqkeyboardmanager").PreviousNextView);

@NgModule({
    bootstrap: [
        AppComponent
    ],
    imports: [
        NativeScriptModule,
        NativescriptBottomNavigationModule,
        AppRoutingModule,
        ApiModule.forConfig(apiConfig),
        CoreModule,
    ],
    declarations: [
        AppComponent,
        HomeComponent,
        MyProfilesComponent,
        EditProfileComponent,
        FieldsComponent,
        AddNewProfileComponent,
        ViewProfileComponent,
        AddNewContactComponent,
        ProfileCardComponent,
        ViewCardComponent
    ],
    entryComponents: [],
    providers: [
        UUIDService,
        ManualExchangeValidatorService,
        PhoneService,
        BeaconExchangeService,
        BeaconScanningService,
        ExchangeTapService,
        BeaconExchangeLogService,
        CommonService,
        ConnectivityService,
        TimerModalService,
        ImageProcessorService,
        DateTimeService,
        RadListViewComponent,
        ProfileDefaultService,
        EventsTracker,
        ErrorsTracker,
        UserSettings,
        LogoutService,
        FormFields,
        MyProfilesDaoService,
        TabSelectionObserverService,
        WebViewService,
        {provide: ErrorHandler, useClass: RavenErrorHandler},
    ],
    schemas: [
        NO_ERRORS_SCHEMA
    ]
})
export class AppModule {
    constructor() {
    }
}

```

**Comment by alexandruantonica** [comment]
@henrychavez Seems like a real problem and is not only in my app. 
I've created a new one in [Playground](https://play.nativescript.org/?template=play-ng&id=Rr37lL) and seems like the problem is the same. 
I only added the plugin and added the module in `app.module.ts`. That's all.

**Comment by alexandruantonica** [comment]
@henrychavez Do you have any news? 
Thanks ;)

**Comment by lukesUbuntu** [comment]
Getting the same issue using nativescript 4.2.3, also did a blank project and just added plugin following example and dies with the following.

`
JavaScript stack trace:
1   _reportError@file:///app/vendor.js:19242:27
2   @file:///app/vendor.js:18715:43
3   forEach@[native code]
4   getNgModuleMetadata@file:///app/vendor.js:18690:56
5   _loadModules@file:///app/vendor.js:32008:70
6   _compileModuleAndComponents@file:///app/vendor.js:31989:48
7   compileModuleAsync@file:///app/vendor.js:31949:64
8   bootstrapModule@file:///app/vendor.js:37439:43
9   _bootstrapper@file:///app/vendor.js:62116:81
10  @file:///app/vendor.js:62169:32
11  notify@file:///app/vendor.js:83514:31
12  @[native code]
13  notifyAppStarted@file:///app/vendor.js:80504:36
14  didFinishLaunchingWithOptions@file:///app/vendor.js:80496:30
15  @[native code]
16  onReceive@file:///app/vendor.js:80394:32
17  UIApplicationMain@[native code]
18  start@file:///app/vendor.js:80636:26
19  run@file:///app/vendor.js:80664:10
20  bootstrapNativeScriptApp@file:///app/vendor.js:62194:26
21  bootstrapApp@file:///app/vendor.js:62128:38
22  bootstrapModule@file:///app/vendor.js:62117:26
23  ./main.ns.ts@file:///app/bun<…>
JavaScript error:
file:///app/vendor.js:19242:27: JS ERROR Error: Unexpected value 'NativescriptBottomNavigationModule' imported by the module 'AppModule'. Please add a @NgModule annotation.
`

**Comment by alexandruantonica** [comment]
Any news, guys ?

**Comment by henrychavez** [comment]
@alexandruantonica  @lukesUbuntu 
The problem was solved, I'll update the plugin in a few hours after finish some test.

**Comment by henrychavez** [comment]
Problem solved in version 1.4.1 please update the plugin. Let me know any issue until that I'll close this issue.
